tox.mysql.ini

```diff
@@ -0,0 +1,32 @@
+[pytest]
+addopts = --reuse-db
+DJANGO_SETTINGS_MODULE = tests.settings
+
+# Add requirements to prevent pytest from discovering test.txt.
+norecursedirs = .* requirements *.egg
+
+[flake8]
+max-line-length = 99
+exclude = *migrations/*
+
+[tox]
+envlist=
+    py27-django{17,18,19}
+    flake8
+
+[testenv]
+deps =
+    -r{toxinidir}/requirements/base.txt
+    -r{toxinidir}/requirements/test.txt
+    django17: Django~=1.7.0
+    django18: Django~=1.8.0
+    django19: Django~=1.9.0
+    djangomaster: https://github.com/django/django/archive/master.tar.gz
```

Comment:
Unless I'm reading this wrong, I don't think we're actually creating any tox test environment with this version of Django.